README.md

```diff
@@ -1,12 +1,40 @@
-# Repo for SRE Content
+# Operate First SRE Content
 
-##  Material
+## Introduction
 
-* [Architecture Decision Records](./ADRs/)
-* [SRE Maturity Model](./sre_maturity.md)
-* [Picking Good SLIs](./picking_good_slis.md)
+This the Operate First repository for Site Reliability documentation. Here we
+intend to collect documentation on the nitty-gritty aspects of building and
+operating an effective Site Reliability organisation. 
+
+## Who runs this repository?
+
+Hello! We're the Red Hat SIG-SRE group, a cross-team group of people inside Red
+Hat with an interest in promoting and developing strong SRE culture. As a
+result you can expect some of the material here to be a little slanted toward
+the way we do things here inside Red Hat, but as we're doing our best to stick
+to what are generally accepted to be good SRE practices you should be able to
+make use of it with, at most, minor modification.
+
+##  What's available here?
```

Comment:
Please add a full list of every doc and a quick blurb on what it is to this page. This will be our "Table of Contents".

Reply:
It would be nice if this was autogenerated, but let's just keep this README in sync with docs as they're merged.

Reply:
As I said in the original PR, autogenerating an index in any useful way is going to require metadata that we don’t have yet (i.e. document category, description, etc)

Reply:
and I've included all the docs that are in the repo.